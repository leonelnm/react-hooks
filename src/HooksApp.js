import React from 'react'


export const HooksApp = () => {
	return (
		<>
			<h1>Hola Hooks!</h1>
		</>
	)
}
