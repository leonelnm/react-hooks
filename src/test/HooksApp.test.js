import '@testing-library/jest-dom'
import React from 'react'
import { shallow } from 'enzyme';
import { HooksApp } from '../HooksApp';

describe('pruebas <HooksApp />', () => {
	
	test('should show component', () => {
		const wrapper = shallow(<HooksApp />)

		expect(wrapper).toMatchSnapshot();
	})
	

})
