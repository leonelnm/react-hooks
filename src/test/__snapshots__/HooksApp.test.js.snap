// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas <HooksApp /> should show component 1`] = `
<Fragment>
  <h1>
    Hola Hooks!
  </h1>
</Fragment>
`;
